config/locales/javascript/javascript.de.yml

```diff
@@ -23,6 +23,7 @@ de:
         no_info: "Zurzeit keine weiteren Informationen verfügbar"
         not_available: "nicht verfügbar"
         offline_since: "offline seit:"
+        blocked: "Blockiert"
```

Comment:
Please don't edit locale files other than for English, all other files are managed by webtranslateit and will be overwritten.

(also, sidenote, it's probably better if you post screenshots here with English, because not everybody understand German, so you don't need to change anything but English anyway.)